packages/survey-creator-core/src/creator-base.ts

```diff
@@ -779,21 +811,31 @@ export class CreatorBase extends Base
   public onGetPageActions: CreatorEvent = new CreatorEvent();
 
   /**
-   * The event is fired when the survey creator is initialized and a survey object (Survey.Survey) is created.
-   *- sender the survey creator object that fires the event
-   *- options.survey  the survey object showing in the creator.
+   * An event that is raised when Survey Creator instantiates a survey for the [Designer](https://surveyjs.io/survey-creator/documentation/end-user-guide#designer-tab) tab. Use this event to customize the survey.
```

Comment:
Please note, if you want this event works from the beginning, please set creator JSON property after you assign a handler to this event. If your creator is empty then set creator JSON with an empty `object: creator.JSON = {}`